java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -2333,7 +2333,7 @@ public TestBuilder timestampSecondsColumn(Long... values) {
 
     private static ColumnVector from(DType type, Object dataArray) {
```

Comment:
When will these APIs be updated to support decimal types?  Had to test if we cannot create a ColumnVector from the data.

Reply:
My initial plan was to put up a small PR to make modifications to DType so that we could handle `scale`  and make sure we don't break any existing tests.  So I had not made any changes in the JNI to support decimal type. I think adding minimal decimal type support(non scalar) and adding corresponding tests makes sense as you said before.